randovania/games/prime1/gui/preset_settings/prime_dock_rando_tab.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+
+import typing
+from typing import TYPE_CHECKING
+
+from PySide6 import QtWidgets
+
+from randovania.games.prime1.layout.prime_configuration import PrimeConfiguration
+from randovania.gui.lib import signal_handling
+from randovania.gui.preset_settings.dock_rando_tab import PresetDockRando
+from randovania.layout.base.dock_rando_configuration import DockRandoMode
+
+if TYPE_CHECKING:
+    from randovania.game_description.game_description import GameDescription
+    from randovania.gui.lib.window_manager import WindowManager
+    from randovania.interface_common.preset_editor import PresetEditor
+    from randovania.layout.preset import Preset
+
+_CHECKBOX_FIELDS = ["blue_save_doors", "blast_shield_lockon"]
+
+
+class PresetPrimeDockRando(PresetDockRando):
+    def __init__(self, editor: PresetEditor, game_description: GameDescription, window_manager: WindowManager):
+        super().__init__(editor, game_description, window_manager)
+
+        self.changes_box = QtWidgets.QGroupBox()
+        self.changes_box.setTitle("Door Changes")
+        self.changes_layout = QtWidgets.QVBoxLayout(self.changes_box)
+
+        extra_widgets: list[tuple[type[QtWidgets.QCheckBox | QtWidgets.QLabel], str, str]] = [
+            (QtWidgets.QCheckBox, "blue_save_doors_check", "Unlock Save Station Doors"),
```

Comment:
Can you add a `,` after the last string in this line, so that all three are formatted the same?